### ISSUE TITLE
rcloader.for outputting object instead of string

### ISSUE THREAD

**Issue by jaylinski** [webpack/jshint-loader#12]
hi there. the loader didn't respect my `.jshintrc`, so i debugged it.
it appears, that the `rcloader.for` function does output an object instead of a string:
```javascript
// console.log(this.resourcePath);
C:\xampp\htdocs\_www\frontend-webpack\src\js\main.js
```
```javascript
// line 34
// var path = rcLoader.for(this.resourcePath);
// console.log(path);
{ '0': 'C',
  '1': ':',
  '2': '\\',
  '3': 'x',
  '4': 'a',
  '5': 'm',
  '6': 'p',
  '7': 'p',
  '8': '\\',
  '9': 'h',
  '10': 't',
  '11': 'd',
  '12': 'o',
  '13': 'c',
  '14': 's',
  '15': '\\',
  '16': '_',
  '17': 'w',
  '18': 'w',
  '19': 'w',
  '20': '\\',
  '21': 'f',
  '22': 'r',
  '23': 'o',
  '24': 'n',
  '25': 't',
  '26': 'e',
  '27': 'n',
  '28': 'd',
  '29': '-',
  '30': 'w',
  '31': 'e',
  '32': 'b',
  '33': 'p',
  '34': 'a',
  '35': 'c',
  '36': 'k',
  '37': '\\',
  '38': '.',
  '39': 'j',
  '40': 's',
  '41': 'h',
  '42': 'i',
  '43': 'n',
  '44': 't',
  '45': 'r',
  '46': 'c' }
```
anyone experiencing a similar issue?
my system:
win7 x64, nodejs 0.10.33
package.json:
"jshint-loader": "^0.8.1",

**Comment by Dashed** [comment]
File issue to https://github.com/spenceralger/rcloader

**Comment by nishp1** [comment]
Looks like this broke in the recent 0.1.4 release of rcloader. Its `for` method now returns parsed file as JSON instead of the filepath.

**Comment by jaylinski** [comment]
@nishp1 good to know i'm not the only one :)

**Comment by jaylinski** [comment]
also see duplicate webpack/jshint-loader/issues/13

**Comment by jaylinski** [comment]
fixed with webpack/jshint-loader/pull/14
